lms/templates/discovery/filter_bar.underscore

```diff
@@ -1,5 +1,4 @@
-<ul class="active-filters facet-list">
-</ul>
-<span>
+<div class="filters-inner">
+    <ul class="active-filters facet-list"></ul>
     <button id="clear-all-filters" class="clear-filters flt-right discovery-button"><%= gettext('CLEAR ALL') %></button>
```

Comment:
@cahrens, thanks for flagging this. I'm not sure there's been a doc review of UI strings. Tagging @srpearce, as she is working on this feature.